src/RequestPage.php

```diff
@@ -96,7 +96,7 @@ function isActionableRequest(AccountRequest &$accountRequest, string $userContex
 	'admin' => 'scratch-confirmaccount-actions'
 ];
 
-function requestActionsForm(AccountRequest &$accountRequest, string $userContext, bool $hasHandledBefore, OutputPage &$output, SpecialPage &$pageContext, &$session) {
+function requestActionsForm(AccountRequest &$accountRequest, string $userContext, bool $hasHandledBefore, OutputPage &$output, SpecialPage &$pageContext, &$session, $timestamp) {
 	global $wgUser;
 	
```

Comment:
Generally avoid calling `getRequest()` more than once.
```suggestion

	$request = $this->getRequest();
```